app/services/checksum_validator.rb

```diff
@@ -2,86 +2,81 @@
 
 # code for validating Moab checksums
 class ChecksumValidator
-  attr_reader :bare_druid, :results, :complete_moab
+  attr_reader :druid, :results, :complete_moab
 
-  alias druid bare_druid
   delegate :moab_storage_root, to: :complete_moab
   delegate :storage_location, to: :moab_storage_root
-  delegate :mark_moab_not_found, :moab, :object_dir, :set_status_as_seen_on_disk, :update_status, to: :moab_validator
-
-  MANIFESTS = 'manifests'
-  MANIFESTS_XML = 'manifestInventory.xml'
-  MODIFIED = 'modified'
-  ADDED = 'added'
-  DELETED = 'deleted'
-  FILES = 'files'
+  delegate :moab, :object_dir, to: :moab_validator
 
   def initialize(complete_moab)
     @complete_moab = complete_moab
-    @bare_druid = complete_moab.preserved_object.druid
-    @results = AuditResults.new(bare_druid, nil, moab_storage_root, 'validate_checksums', logger: Audit::Checksum.logger)
+    @druid = complete_moab.preserved_object.druid
+    @results = AuditResults.new(druid, nil, moab_storage_root, 'validate_checksums', logger: Audit::Checksum.logger)
     # TODO: fix fragile interdependence, MoabValidator wants AuditResults instance, but we want MoabValidator#moab.current_version_id
     # in that AuditResults instance.  so set AuditResults#actual_version after both instances have been created.
     @results.actual_version = moab.current_version_id
   end
 
   def validate_checksums
     # check first thing to make sure the moab is present on disk, otherwise weird errors later
-    return results.result_array unless check_file_existence!
+    return persist! { moab_validator.mark_moab_not_found } unless moab_exists?
```

Comment:
in the validate_checksums method, which still has a high cognitive load, would it be clearer as "if not" rather than "unless" ?   Or perhaps have a method named for the opposite case (ew)  `moab_absent?`

Reply:
Since this is only used in one place, I changed to `moab_absent?`.